src/components/Person.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Person.scss';
+
+function marriage(props) {
+  if ((props.props.sex === 'm') && (props.props.isMarried)) {
+    return (`My wife's name is ${props.props.partnerName}`);
+  }
+
+  if ((props.props.sex === 'f') && (props.props.isMarried)) {
+    return (`My husband's name is ${props.props.partnerName}`);
+  }
+
+  return ('I am not married');
```

Comment:
Don't repeat yourself, you change only one word in two if conditions
```suggestion
  const partnerSex = sex === 'm' ? 'wife' : 'husband';
  
  if (isMarried) {
    return (`My ${partnerSex}'s name is ${partnerName}`);
  }
  
  return ('I am not married');
```

Reply:
Also use destructuring for getting main variables
And check the name of the function, it should be understandable, for example: getMarriageStatus